static/scenes/speedsketch/js/views/cardsView.js

```diff
@@ -224,44 +224,42 @@ app.view.CardsView.prototype.fetchAndShowDrawingNeighbors = function(round) {
       console.error(jqXHR, textStatus, errorThrown);
     })
     .done((data) => {
-      if (data[0] == "SUCCESS") {
```

Comment:
Looks like we're just removing this part.

Reply:
Yes, just removing the test for the string "SUCCESS" as it is not part of the OnePlatform recognition response.